### ISSUE TITLE
Tier data sync from http://www.borischen.co/ [Feature Request..of sorts]

### ISSUE THREAD

**Issue by b0bbybacc** [jayjzheng/ff_rankings#14]
I noticed on Boris's site he has a blog post that he is now updating tier data every hour. I used your tool going through a mock draft today and noticed two things:

1. http://jayzheng.com/ff/#/draft-aid says last updated Aug 19th, 2015
2. As an example Kelvin Benjamin is showing as a tier 6 WR option but was recently injured and is out for the season. This makes sense if the tool on your end last pulled data from Boris's site 2/3 days ago since I believe he was just injured around that time. 

When you look at Boris's current WR tier data as of today Benjamin is no where to be found which makes sense since he is now out for the year and Boris is polling fantasypros every hour.

TL;DR now that Boris has committed to updating tier information based on FantasyPros ECR (polled fresh every hour) and running it through his tier alg. is there a way for you to poll new tier information from him site (and new ranking information from fantasypros) a couple of times a day?

**Comment by jayjzheng** [comment]
it is certainly doable. the current update is done manually, and I thought about automating the process.

However, it will probably need to wait as I am currently working on something to let users upload their own rankings.
